CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.0.0
+
+- Deprecated the list based versions of `TenantSecurityKMSClient.decryptBatch` and `TenantSecurityKMSClient.encryptBatch` in favor of a Map based interface that allows for partial failure.
```

Comment:
```suggestion
- Deprecated the list based versions of `TenantSecurityKMSClient.decryptBatch` and `TenantSecurityKMSClient.encryptBatch` in favor of a Map based interface that allows for partial failure. NOTE: This version of the client will only work with versions 2.x of the Tenant Security Proxy container.
```

Should the title of this also be `2.0.0-beta` or something? Maybe we don't actually need to release this as a beta at all?

Reply:
I'm releasing it as a SNAPSHOT, the changelog could be updated if we add anything to it.